docs/mosviz/displayspectra.rst

```diff
@@ -2,7 +2,14 @@
 Display Spectra
 ***************
 
-Coming soon
+Spectra loaded by the user will be automatically displayed in the viewers, with 2d spectra
```

Comment:
"The spectra linked to the source selected in the table viewer will be automatically..."